elements/lisk-p2p/src/p2p.ts

```diff
@@ -317,6 +319,18 @@ export class P2P extends EventEmitter {
 			if (this._isNetworkReady()) {
 				this.emit(EVENT_NETWORK_READY);
 			}
+
+			// Once we successfuly connected to a SeedPeer we try to make new connection or initiate eviction from it LIP-0004
+			const isSeedPeer = this._sanitizedPeerLists.seedPeers.find(
+				peer => peer.peerId === peerInfo.peerId,
+			);
+
+			if (isSeedPeer) {
```

Comment:
So when we successfully connect to a `seedPeer`,  we should not initiate the discovery from others. When we make an outbound connection we `fetchPeers` as part of `connect` event so that's enough to cover discovery from seed

Reply:
Seems this part could be removed.